content/contribute/design-documents/index.html

```diff
@@ -190,7 +190,7 @@ <h3 id="beam-model">Beam Model</h3>
   <li>Error Reporting Callback (WIP) [<a href="https://docs.google.com/document/d/1o2VXwCL97k3G-1BR9RSKNc6XtJTIA6SEKPMne91S67Y">doc</a>]</li>
   <li>Snapshotting and Updating Beam Pipelines [<a href="https://docs.google.com/document/d/1UWhnYPgui0gUYOsuGcCjLuoOUlGA4QaY91n8p3wz9MY">doc</a>]</li>
   <li>Requiring PTransform to set a coder on its resulting collections [<a href="https://lists.apache.org/thread.html/1dde0b5a93c2983cbab5f68ce7c74580102f5bb2baaa816585d7eabb@%3Cdev.beam.apache.org%3E">mail</a>]</li>
-  <li>[PROPOSAL] @RequiresStableInput [<a href="https://docs.google.com/document/d/117yRKbbcEdm3eIKB_26BHOJGmHSZl1YNoF0RqWGtqAM">doc</a>] [<a href="https://lists.apache.org/thread.html/ae3c838df060e47148439d1dad818d5e927b2a25ff00cc4153221dff@%3Cdev.beam.apache.org%3E">mail</a>]</li>
+  <li>[PROPOSAL] @RequiresStableInput [<a href="https://docs.google.com/document/d/117yRKbbcEdm3eIKB_26BHOJGmHSZl1YNoF0RqWGtqAM">doc</a>], [<a href="https://lists.apache.org/thread.html/ae3c838df060e47148439d1dad818d5e927b2a25ff00cc4153221dff@%3Cdev.beam.apache.org%3E">mail</a>]</li>
```

Comment:
I believe PR should not contain changes to `content` directory since merging that is tricky.
Could you run a command `git checkout -- content` to remove autogenerated content and amend your commit to exclude files from `content` directory?

Reply:
OK I see. Now I know the structure of the repo. Thank you for all the comments!